### DISCUSSION TITLE
403 invalid signature when click link from email for forgot password

### DISCUSSION THREAD

**Discussion by febber80** [nginx/nginx#399] [General]
i have project laravel in sub directory, when i send link of forget password into email and when i clicked the link  an error appeared 403 invalid signature, 

this is my .env file :
`APP_URL=https://mydomain.com/project`

and this is my nginx config :
```
location /project{
     alias /www/html/project/public;
     index index.php index.html default.php 
     try_files $uri $uri/ @project;
  location @project{
      rewrite /project/(.*)$ /project/index.php?$1 last;
    }
```
can anyone help ? thanks before

**Comment by Maryna-f5** [comment]
This discussion has been inactive for more than 90 days, closing it.
